src/gleam/map.gleam

```diff
@@ -465,7 +524,7 @@ if javascript {
 /// ```
 ///
 pub fn drop(from map: Map(k, v), drop disallowed_keys: List(k)) -> Map(k, v) {
-  list.fold(over: disallowed_keys, from: map, with: delete)
+  do_drop(map, disallowed_keys)
```

Comment:
No need for a `do_` function here, this function can recurse

Reply:
Hmm ... good point. Now that to told me, I see it ;-)